products/pop_os.md

```diff
@@ -24,27 +24,37 @@ releases:
     eol: 2027-04-25
     link: https://blog.system76.com/post/popos-2204-lts-has-landed
 
+    latestReleaseDate: 2022-04-25
```

Comment:
Strange, I disabled the auto configuration in https://github.com/endoflife-date/endoflife.date/commit/65bc96b07232ed9cf2ebb5711cc02a0f3cbf8608 so it should not have been updated.

Reply:
The code isn't aligned with this (I've made the same mistake earlier).

The `latest.py` script just does file matches against the [releases directory](https://github.com/endoflife-date/release-data/tree/main/releases). The `auto` configuration is only picked up by the `distrowatch.py` script, which regenerates the corresponding JSON file.

So as long as this file exists: https://github.com/endoflife-date/release-data/blob/main/releases/pop_os.json, it will keep getting used.

`latest.py` doesn't read any of the markdown files for configuration, it merely writes to it.

Reply:
I've deleted it for now: https://github.com/endoflife-date/release-data/tree/main/releases

Will undo the popOS changes.

Reply:
popOS changes are still in this PR. Let me do it so I can merge this PR.

Reply:
#2059 fixed this up already. Can merge now.